.github/workflows/enos-run.yml

```diff
@@ -78,10 +78,9 @@ jobs:
         include:
           - filter: 'e2e_aws builder:crt'
           - filter: 'e2e_database'
-          - filter: 'e2e_base builder:crt'
-          - filter: 'e2e_base_with_vault builder:crt'
-          # - filter: 'e2e_ui builder:crt' # Don't run UI tests yet. takes too long.
-    runs-on: ${{ fromJSON(vars.RUNNER) }}
+          - filter: 'e2e_docker_base builder:crt'
+          - filter: 'e2e_docker_base_with_vault builder:crt'
+    runs-on: ${{ fromJSON(vars.RUNNER_LARGE) }}
```

Comment:
This change was needed to mitigate intermittent failure we were seeing in the past when running these docker scenarios in CI. We would often get errors when tests were connecting to targets. Changing to a larger runner instance prevents these errors from occurring.